### PR TITLE
Unfuzz available results

### PR DESCRIPTION
**Pull request by jordanwilking**
Unfuzzes the available results. 

Testing:
- Verify that the results are unfuzzed (exact counts rather than fuzzy counts)

image
image

### CODE REVIEW

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/component/result-selector/query-feed.tsx [hunk 1]
```diff
@@ -103,7 +103,7 @@ const QueryStatusRow = ({ status, query }: { status: Status; query: any }) => {
       </Cell>
       <Cell data-id="available-label">
         <CellValue
-          value={fuzzyHits(status.count)}
+          value={`${status.count} hit${status.count === 1 ? '' : 's'}`}
```
Comment (SmithJosh) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/component/result-selector/query-feed.tsx:106]: +1 for correct pluralization

**Review by SmithJosh — APPROVED** [review]
So was the conclusion that we don't need to hide the available results count since the user can count them via paging anyway?

**Review by vinamartin — APPROVED** [review]

**Review by andrewkfiedler — APPROVED** [review]

**Review by austinsteffes-aus — APPROVED** [review]

**Comment by jordanwilking** [comment]
> So was the conclusion that we don't need to hide the available results count since the user can count them via paging anyway?

@SmithJosh, I wasn't in that meeting, but I imagine that's part of it. Obfuscating the available results doesn't hide much and it makes that field pretty worthless in most cases.

**Review by SmithJosh — APPROVED** [review]
Hero successful. Verified that the "Available" stat shows the exact hit count while "Possible" is fuzzy.

Also, as an aside, imo the aggregate hit count would be more helpful if it showed the available hits
Screen Shot 2020-12-18 at 9 40 37 AM

**Comment by jordanwilking** [comment]
image

**Pull request merged**